templates/cinder/bin/init.sh

```diff
@@ -26,51 +26,54 @@ export DBPASSWORD=${DatabasePassword:?"Please specify a DatabasePassword variabl
 export PASSWORD=${CinderPassword:?"Please specify a CinderPassword variable."}
 export TRANSPORTURL=${TransportURL:-""}
 
-export CUSTOMCONF=${CustomConf:-""}
-
+DEFAULT_DIR=/var/lib/config-data/default
+CUSTOM_DIR=/var/lib/config-data/custom
+MERGED_DIR=/var/lib/config-data/merged
 SVC_CFG=/etc/cinder/cinder.conf
-SVC_CFG_MERGED=/var/lib/config-data/merged/cinder.conf
+SVC_CFG_MERGED=${MERGED_DIR}/cinder.conf
+SVC_CFG_MERGED_DIR=${MERGED_DIR}/cinder.conf.d
+
+mkdir -p ${SVC_CFG_MERGED_DIR}
 
-# expect that the common.sh is in the same dir as the calling script
-SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
-. ${SCRIPTPATH}/common.sh --source-only
+cp ${DEFAULT_DIR}/* ${MERGED_DIR}
 
 # Copy default service config from container image as base
 cp -a ${SVC_CFG} ${SVC_CFG_MERGED}
```

Comment:
Without this line, the resulting /etc/cinder directory in the service containers will have *no* cinder.conf. That may be OK if we assume the file is basically a bunch of comments that document every config setting's default value. Still, I see value in adding it to the containers, if for no other reason than it may help people find options that they want to modify when they're debugging a live problem.

Reply:
I agree that has value, moreover now that we are transparent with the config options in the installer.
Maybe we should rename it to something like `cinder.conf.sample` to indicate the intent of the file and avoid misunderstandings.